models/export.js

```diff
@@ -6,27 +6,34 @@ if (Meteor.isServer) {
   // `ApiRoutes.path('boards/export', boardId)``
   // on the client instead of copy/pasting the route path manually between the
   // client and the server.
-  /*
-   * This route is used to export the board FROM THE APPLICATION.
-   * If user is already logged-in, pass loginToken as param "authToken":
-   * '/api/boards/:boardId/export?authToken=:token'
+  /**
+   * @operation export
```

Comment:
Hmm, there is something wrong here.
I forgot to re-run the openAPI python script, and it seems the `export` call is not generated...

Reply:
OK, pushed a new commit to openapi that fixes that